Services/sources/TMU.c

```diff
@@ -0,0 +1,544 @@
+/*
+ * TMU.c
+ *
+ * Created: 2/24/2020 11:47:28 AM
+ *  Author: Reem
+ */
+#include "TMU.h"
+
+/*******Static Global Variables*********/
+static TMU_Cfg gstr_TMUcfg;
+static Timer_cfg_s gstr_TimerCfg;
+static TMU_TASK_Cfg *gstr_TasksBuffer[TMU_TASK_BUFFER_SIZE];
```

Comment:
Where the config struct ? If it in the .h file so the question here do we need to keep it visable to the user ?